### PR TITLE
WIP: add external CCM for Nutanix

### PR DESCRIPTION
**Pull request by wolfganghuse**
What this PR does / why we need it:
Support for external CCM for Nutanix

What type of PR is this?
/kind feature

Does this PR introduce a user-facing change? Then add your Release Note here:

```release-note
Added support for Nutanix external CCM
```

Documentation:

```docs
NONE
```

### CODE REVIEW

**Comment by xmudrii** [comment]
/release-note-edit
```release-note
Added support for Nutanix external CCM
```

**Comment by xmudrii** [comment]
/ok-to-test
/docs-none
/assign
I'll take a look at this PR soon

File: addons/ccm-nutanix/ccm-nutanix.yaml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+              cpu: 100m
+              memory: 50Mi
+          volumeMounts:
+            - mountPath: /etc/cloud
+              name: nutanix-config-volume
+              readOnly: true
+      volumes:
+        - name: nutanix-config-volume
+          configMap:
+            name: nutanix-config
```
[the remark this answers is not in the document]
Comment (wolfganghuse) [reply addons/ccm-nutanix/ccm-nutanix.yaml]: Done

**Comment by kron4eg** [comment]
@wolfganghuse this PR was used as a base for #3034. Thanks for inspiration!

/close

**Pull request closed without merging**